src/store/sagas/setupAndAuth.ts

```diff
@@ -55,10 +66,20 @@ function* setupWalletWorker( { payload } ) {
     },
     version: DeviceInfo.getVersion()
   }
-
+  const wallet: Wallet = {
+    walletId,
+    walletName,
+    userName: walletName,
+    security,
+    primaryMnemonic: '',
```

Comment:
We don't need the primary mnemonic and the primary seed to be initialized as empty strings. We can leave those parameters out(as they've already been operationalized in the interface)

Reply:
Resolved